apis/actions.md

```diff
@@ -119,10 +126,8 @@ var button = new Gtk.Button.from_icon_name ("process-stop", Gtk.IconSize.LARGE_T
 };
 ```
 
-It may now be clear why we needed to declare our action at the beginning of `activate ()`: before we can get a list of the accelerators associated with the action, we have to define those accelerations and add them to the Application.
-
 Compile your app one last time and hover over the HeaderBar Button to see its description and associated keyboard shortcuts.
 
 {% hint style="info" %}
-If you're having trouble, you can view the full example code [here on GitHub](https://github.com/vala-lang/examples/tree/glib-action)
+If you're having trouble, you can view the full example code [here on GitHub](https://github.com/vala-lang/examples/tree/glib-action).  You can learn more about Actions from GNOME [developer documentation](https://developer.gnome.org/documentation/tutorials/actions.html)
```

Comment:
I'm not sure if it's OK to link to GNOME's documentation, so I added these references (here and in the _Launchers_ page) in a separate commit so that they can be easily removed.

Reply:
This makes sense! I switched the link (and wording around it) to valadoc reference documentation.